_posts/2016-03-02-Rust-1.7.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Announcing Rust 1.7"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.7. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.7][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.7][notes] on GitHub.
+About 1300 patches were landed in this release.
+
+[install]: http://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-170-2016-03-03
+
+### What's in 1.7 stable
+
+This release contains a number of small refinements, one major feature, and
+a change to [Crates.io](https://crates.io).
+
+#### New Language features
+
+... there were none! While we have several things cooking for future releases,
+the timeframe in which 1.7 included the holidays, which means less time for
```

Comment:
not sure if we need an excuse here. Maybe just "for future releases, none of them landed for the 1.7 cutoff."